fe2-android/app/src/main/java/com/github/dedis/popstellar/model/qrcode/MainPublicKeyData.java

```diff
@@ -0,0 +1,32 @@
+package com.github.dedis.popstellar.model.qrcode;
+
+import com.github.dedis.popstellar.model.network.serializer.JsonUtils;
+import com.google.gson.Gson;
+import com.google.gson.annotations.SerializedName;
+
+public class MainPublicKeyData {
```

Comment:
Is there a reason this does not have `@Immutable`?

Reply:
Yeah I was doing a little review of my own and came accross this 😁